packs/equipment/the-trumpet.json

```diff
@@ -9,8 +9,8 @@
+            "gm": "<p><strong>Epitome</strong> Heralds outfitted with trumpets take up position on the battlements of Harrowheart. These heralds use their trumpets to raise the alarm should intruders enter the Harrow Court (an event that won't happen in this adventure—but that shall in the next one!). The heralds know the formulas for all manner of magical instruments, including horns of blasting, horns of fog, and maestro's instruments, and can teach these formulas to the PCs.</p>",
+            "value": "<p><strong>Suit</strong> Shield (Con)</p>\n<p><strong>Alignment</strong> lawful good</p><hr /><p><em>The Trumpet</em> represents timely warning. As long as you have <em>The Trumpet</em> invested, you (and only you) hear the sound of trumpets when you're about to be attacked. This grants you a +2 item bonus to initiative checks made with hearing-based Perception checks. This bonus increases to +3 if you're at least 17th level.</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> envision</p>\n<p><strong>Frequency</strong> once per day</p><hr /><p><strong>Effect</strong> A blast of inspiring noise erupts from the card in a @Template[cone|distance:60]. Creatures in the area take @Damage[14d6[sonic]|options:area-damage] damage (with a @Check[fortitude|against:class-spell|basic|options:area-effect] save equal to your class DC or spell DC). Creatures that fail the save are @UUID[Compendium.pf2e.conditionitems.Item.Deafened] for 1 round (1 minute on a critical failure). The damage increases to @Damage[16d6[sonic]|options:area-damage] if you're at least 13th level, to @Damage[18d6[sonic]|options:area-damage] if you're at least 15th level, to @Damage[20d6[sonic]|options:area-damage] if you're at least 17th level, and to @Damage[22d6[sonic]|options:area-damage] if you're at least 19th level.</p>"
```

Comment:
Consider making this damage auto-level.

Reply:
I did consider it, but doing so would use the level of the item when not on an actor, and since it needs to use a `max` to have the correct damage dice when it's on a low level actor, it means it will display the wrong damage when looked at outside of an actor